c7n/resources/ebs.py

```diff
@@ -32,7 +32,7 @@
 from c7n.manager import resources
 from c7n.resources.kms import ResourceKmsKeyAlias
 from c7n.query import QueryResourceManager, TypeInfo
-from c7n.tags import Tag
+from c7n.tags import Tag, coalesce_copy_user_tags
```

Comment:
Interesting, I did not know that function was in there. Good call, it seems to be perfect for this!